src/mmw/js/src/analyze/views.js

```diff
@@ -109,7 +109,7 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
 
         this.$el.animate({ height: '55%' }, 200, function() {
             self.trigger('animateIn');
-            App.map.set('halfSize', true);
+            App.map.setHalfSize(true);
```

Comment:
I wrapped it up together in a single call & attribute because I didn't want the fit call to happen during an animation and wanted the map view to be able to handle the transition internally - but I see that the code never accounted for that anyway, and just pops a class on.  I still kind of like having the single call so that it could manage that, which I now suspect is the cause of #518